schemas/query-subscription-consumer.schema.json

```diff
@@ -0,0 +1,77 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$ref": "#/definitions/Main",
+  "definitions": {
+    "Main": {
+      "title": "Subscription Wrapper Schema",
```

Comment:
Please see https://github.com/getsentry/sentry-kafka-schemas#using-python-types for how the `title` property should be used

Reply:
I don't think I quite understand what I need to do from reading that documentation. Does every property need a title? Also I was a bit confused by the path listed, imo it should say `sentry-kafka-schemas/python/sentry_kafka_schemas/schema_types/`.

Reply:
Just the top level one.. and optionally anything else that you wish to generate a Python TypedDict for.